api/apps/articles/model/schema.coffee

```diff
@@ -75,7 +75,10 @@ denormalizedArtwork = (->
   thumbnail_title: @string().allow('', null)
   thumbnail_teaser: @string().allow('', null)
   thumbnail_image: @string().allow('', null)
+  vertical: @array().items(@string()).default([])
```

Comment:
Yep, this is something to discuss. I was told that Vertical and Subverticals won't change often which gives us an opportunity to optimize for query speeds. If we were to validate an objectId instead, the article would also have to fetch a Curation each time the article got rendered. Maybe this won't be so bad with GraphQL though. 

My feeling is either it will be simple like
```
@array().items(@string()).allow('Art', 'Visual Culture', 'Market').default([])
```
where we allow certain vertical strings and make the front end admin match this, or
```
@array().items(
      @object().keys
          id: @objectId()
          name: @string().allow('', null)
    ).allow(null).default([])
```
where we denormalize some of the data that makes it easier for display article info without aggregating fetches.